src/standard.rs

```diff
@@ -130,6 +152,12 @@ pub trait StandardDocumentOps: AlloyClient {
         plaintext_document: PlaintextDocumentWithEdek,
         metadata: &AlloyMetadata,
     ) -> Result<EncryptedDocument, AlloyError>;
+    /// TODO
```

Comment:
TODO in another linked ticket or still in this one?

Reply:
This one. Done